storage/innobase/log/log0recv.cc

```diff
@@ -1020,13 +1020,19 @@ static void recv_apply_log_rec(recv_addr_t *recv_addr) {
     return;
   }
 
-  bool found;
   const page_id_t page_id(recv_addr->space, recv_addr->page_no);
 
-  const page_size_t page_size =
-      fil_space_get_page_size(recv_addr->space, &found);
+  fil_space_t *space = fil_space_acquire_for_io_with_load(recv_addr->space);
```

Comment:
fil_space_get_size + fil_space_get() should work?

Reply:
because tablespace cannot be dropped concurrently during redo ?